src/applications/disability-benefits/all-claims/tests/config/submitForm.unit.spec.js

```diff
@@ -3,9 +3,23 @@ import sinon from 'sinon';
 
 import formConfig from '../../config/form';
 
+function mockStorage() {
+  const storage = {};
+  return {
+    getItem(key) {
+      return storage[key];
+    },
+  };
+}
```

Comment:
This will prevent the test from breaking when it calls `localStorage.getItem()`, but I don't think it is doing much else.  Additionally, these tests aren't setting the token in localStorage and checking to see that it is part of the request header - should that be done?

If we don't really care about what is going in and out of `localStorage`, we could just stub it out instead of implementing some of the functions. The [Dashboard App tests](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/personalization/dashboard/tests/containers/DashboardApp.unit.spec.jsx#L20) seem like a decent example:
![image](https://user-images.githubusercontent.com/2008881/78056362-dc070880-7339-11ea-95a8-1843cfd70d2e.png)

Reply:
Testing the token is not necessary